### ISSUE TITLE
add binaries only for specific platform

### ISSUE THREAD

**Issue by breuerfelix** [breuerfelix/chromedriver-py#7]

**Comment by breuerfelix** [comment]
@KotlinIsland yes that would reduce the pypi package size for each distro about 1/3 of the size.

**Comment by KotlinIsland** [comment]
if there are 5 binaries, wouldn't the wheel be 1/5 the size?

**Comment by breuerfelix** [comment]
> if there are 5 binaries, wouldn't the wheel be 1/5 the size?

depends on how good the compression currently is. maybe you are right, we have to see :)
